### PR TITLE
Bluetooth node

### PR DESCRIPTION
**Pull request by vegkams**
Node for reading strings broadcast over bluetooth and publishing the received string as a ROS topic.

### CODE REVIEW

File: bluetooth/scripts/hc-05_interface.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    def read_bluetooth_data(self):
+        while not rospy.is_shutdown():
+            BT_msg = self._readline()
+            if len(BT_msg) > 0:
+                self.pub_bluetooth.publish(BT_msg)
+            rospy.Rate(10).sleep()
+
+    def _readline(self):
+        eol = b'\r'
+        leneol = len(eol)
```
Comment (sindrehan) [inline bluetooth/scripts/hc-05_interface.py]: `leneol` is unnecessary.

**Review by sindrehan — CHANGES_REQUESTED** [review]
Remove the unnecessary length variable and I am pleased.

File: bluetooth/launch/bluetooth.launch [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<launch>
+<env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time}] [${node}]: ${message}" />
+<node name="bluetooth" pkg="bluetooth" type="hc-05_interface.py" output="screen"/>
+</launch>
```
Comment (mortenfyhn) [inline bluetooth/launch/bluetooth.launch]: Add newline at end of file

File: bluetooth/scripts/hc-05_interface.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            'bluetooth/container_id',
+            ContainerID,
+            queue_size=10)
+
+    def read_bluetooth_data(self):
+        while not rospy.is_shutdown():
+            BT_msg = self._readline()
+            if len(BT_msg) > 0:
+                self.pub_bluetooth.publish(BT_msg)
+            rospy.Rate(10).sleep()
```
Comment (mortenfyhn) [inline bluetooth/scripts/hc-05_interface.py]: Maybe make spin rate a "constant" outside the class?

File: vortex_msgs/msg/ContainerID.msg [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Container ID read over bluetooth
+# Format of the message is "xxxxxxx\r", where x is a digit
+string container_id
```
Comment (mortenfyhn) [inline vortex_msgs/msg/ContainerID.msg]: Newline

**Review by mortenfyhn — CHANGES_REQUESTED** [review]
Looks good, but look at the comments before merging

**Comment by sindrehan** [comment]
[Bluetooth-node](https://trello.com/c/ub8XMAn7/175-bluetooth-node)

**Review by mortenfyhn — APPROVED** [review]
Looks good to me

**Review by sindrehan — APPROVED** [review]
Very good.

**Pull request merged**